cmd/composer-cli/projects/info.go

```diff
@@ -18,39 +19,69 @@ var (
 		Short: "Show detailed info about the listed projects",
 		Example: `  composer-cli projects info tmux
   composer-cli projects info tmux --json
-  composer-cli projects info tmux --distro fedora-38`,
+  composer-cli projects info tmux --distro fedora-38
+  composer-cli projects info tmux --distro fedora-38 --arch aarch64`,
 		RunE: info,
 		Args: cobra.MinimumNArgs(1),
 	}
 )
 
 func init() {
-	infoCmd.Flags().StringVarP(&distro, "distro", "", "", "Return results for distribution")
+	infoCmd.Flags().StringVarP(&distro, "distro", "", "", "Distribution")
+	infoCmd.Flags().StringVarP(&arch, "arch", "", "", "Architecture")
 	projectsCmd.AddCommand(infoCmd)
 }
 
 func info(cmd *cobra.Command, args []string) error {
 	names := root.GetCommaArgs(args)
 
-	projects, resp, err := root.Client.ProjectsInfo(names, distro)
-	if err != nil {
-		return root.ExecutionError(cmd, "Info Error: %s", err)
-	}
-	if resp != nil && !resp.Status {
-		return root.ExecutionErrors(cmd, resp.Errors)
-	}
+	var err error
+	if root.Cloud.Exists() {
+		if len(distro) == 0 {
+			distro, err = common.GetHostDistroName()
+			if err != nil {
+				return root.ExecutionError(cmd, "Error determining host distribution: %s", err)
+			}
+		}
 
-	for _, p := range projects {
-		root.PrintWrap(6, 80, fmt.Sprintf("Name: %s", p.Name))
-		root.PrintWrap(9, 80, fmt.Sprintf("Summary: %s", p.Summary))
-		root.PrintWrap(10, 80, fmt.Sprintf("Homepage: %s", p.Homepage))
-		root.PrintWrap(13, 80, fmt.Sprintf("Description: %s", p.Description))
-		fmt.Println("Builds: ")
-		for _, b := range p.Builds {
-			fmt.Println("    ", b)
+		if len(arch) == 0 {
+			arch = common.HostArch()
+		}
+
+		packages, err := root.Cloud.SearchPackages(names, distro, arch)
+		if err != nil {
+			return root.ExecutionError(cmd, "Info Error: %s", err)
 		}
-		fmt.Printf("\n\n")
-	}
 
+		for _, p := range packages {
+			root.PrintWrap(6, 80, fmt.Sprintf("Name: %s", p.Name))
```

Comment:
why 80? or just because it always was ment to be used on 80x25 monitors :-D

Reply:
Because trying to get everyone to agree on a width is impossible ;) And it's what was used before.

Reply:
… or just don't wrap and let the terminal do it :-D 
just kidding - LGTM!